tso_session/intro_TSO_data_pipeline.ipynb

```diff
@@ -136,11 +136,74 @@
     "First the imports.\n"
```

Comment:
<p>Please move these imports to the top of the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>